docs/running-on-yarn.md

```diff
@@ -304,6 +304,14 @@ If you need a reference to the proper location to put log files in the YARN so t
   </td>
 </tr>
 <tr>
+  <td><code>spark.yarn.attemptFailuresValidityInterval</code></td>
+  <td>-1</td>
+  <td>
+  Ignore the failure number which happens out the validity interval (in millisecond).
```

Comment:
After reading YARN-611, I think this needs a better explanation, actually. Something along the lines of:

"Defines the validity interval for AM failure tracking. If the AM has been running for at least this long, the AM failure count will be reset."